### PR TITLE
feat: sign in screen - mobile responsiveness bb-61

### PR DESCRIPTION
**Pull request by ihorLysak**
### Summary
Implemented mobile responsiveness for sign in screen component

https://github.com/user-attachments/assets/96f1e660-a78b-407c-b4b4-9cef49aa9ffc

### Changes
- created 4th circle, added it to SignInForm component and added a separate css class for it.

### CODE REVIEW

File: apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css [hunk 1]
```diff
@@ -120,12 +120,71 @@
 	border-radius: 50%;
 }
 
+.circle-gradient4 {
+	position: absolute;
+	display: none;
+	background: linear-gradient(to right, #0085ff 0%, #00f0ff 100%);
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css:126]: Do we have these colors in variables.css?

File: packages/shared/src/modules/users/libs/validation-schemas/user-sign-up.validation-schema.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ const userSignUp = z
 		email: z
 			.string()
 			.trim()
-			.min(UserValidationRule.EMAIL_MINIMUM_LENGTH, {
+			.min(UserValidationRule.EMAIL_REQUIRE, {
```
Comment (fshabanov) [inline packages/shared/src/modules/users/libs/validation-schemas/user-sign-up.validation-schema.ts:16]: This shouldn't have changed

**Review by fshabanov — CHANGES_REQUESTED** [review]
Could you add a video demonstrating different screen sizes?

File: apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css (hunk 1 above)
Comment (ihorLysak) [reply apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css:126]: We weren't at the time when I written this, will use css variables now

File: packages/shared/src/modules/users/libs/validation-schemas/user-sign-up.validation-schema.ts (hunk 2 above)
Comment (ihorLysak) [reply packages/shared/src/modules/users/libs/validation-schemas/user-sign-up.validation-schema.ts:16]: UserValidationRule.EMAIL_REQUIRE does not exist on the main branch

**Review by fshabanov — APPROVED** [review]

**Pull request merged**